detect/synthetic-monitoring/playwright-monitoring-comparison.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Playwright Monitoring Comparison'
```

Comment:
```suggestion
title: 'Playwright Monitoring Options'
```

I feel that the page first explains the options, helps make a choice, then compares with the table? I'm not too settled on this one; it's tricky for sure.